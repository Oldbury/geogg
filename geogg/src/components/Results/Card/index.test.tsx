import React from 'react'
import renderer from 'react-test-renderer'
import ResultCard from '.'

test('Search result card renders', () => {

    const component = renderer.create(
        <ResultCard name="Test" /> 
    )

    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()

})