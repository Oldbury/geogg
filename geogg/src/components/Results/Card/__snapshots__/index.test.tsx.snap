// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search result card renders 1`] = `
<div
  className="bg-white border-b px-3 py-2  text-black"
>
  <h1>
    Test
  </h1>
</div>
`;
